File: samples_generated_code/samples_1/athene/baseline_code_c/code_row_107.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

typedef struct {
    char *name;
    int status;
} Person;

int main() {
    Person *p = (Person *)malloc(sizeof(Person));
    if (p != NULL) {
        p->status = 0;
        p->name = malloc(5);
        if (p->name != NULL) {
            strcpy(p->name, "John");
        }
    }
    free(p->name);
    free(p);
    return 0;
}

File: samples_generated_code/sample_1/athene/baseline_code_java/code_row_107.java
import java.util.Arrays;

public class Person {
    int status;
    String name;

    public static void main(String[] args) {
        Person p = new Person();
        p.status = 0;
        p.name = "John";
        System.out.println("Status: " + p.status + ", Name: " + p.name);
    }
}